sphinx_rtd_theme/layout.html

```diff
@@ -110,7 +109,7 @@
 
           {% if theme_display_version %}
             {%- set nav_version = version %}
-            {% if READTHEDOCS and current_version %}
+            {% if (READTHEDOCS or versioning) and current_version %}
```

Comment:
This is not going to work. All variables set in the theme are prepended with `theme_`.

Reply:
Fixed and tested now.